src/Compilers/VisualBasic/Test/Syntax/Parser/ParserRegressionTests.vb

```diff
@@ -934,4 +934,15 @@ End Enum
         Assert.Equal(Syntax.InternalSyntax.Scanner.BadTokenCountLimit, tree.GetDiagnostics().Where(Function(d) d.Code = ERRID.ERR_IllegalChar).Count())
     End Sub
 
+    <Fact, WorkItem(48587, "https://github.com/dotnet/roslyn/issues/48587")>
+    Public Sub ParseTrailingTextAfterPropertyWithParentheses()
+        ParseAndVerify(<![CDATA[
+Class C
+    Public ReadOnly Property NumberOfResult() String Integer JohnDoe WwwIIWww Wow
```

Comment:
Can you add some variants on this test where we have the possible resync points? Ie, some garbage before the `As`, `=`, or `Implements` keywords.

Reply:
@333fred Thanks. I extended the test.